pkgs/misc/drivers/em100/default.nix

```
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, curl
+, pkgconfig
```

```suggestion
, pkg-config
```

pkgconfig is a alias to pkg-config